codebuild/create_project.py

```diff
@@ -319,20 +334,50 @@ def main(**kwargs):
         except exceptions.NoCredentialsError:
             raise SystemExit(f"Something went wrong with your AWS credentials;  Exiting.")
 
-        if args.dry_run:
-            logging.info('Respecting dry-run flag.  Done')
+        if args.production:
+            if args.modify_existing is True:
+                stack_name = config.get("Global", "stack_name")
+                change_set_name = "A" + hashlib.sha256(bytes(time.asctime().encode('utf-8'))).hexdigest()
+                # ^^^ Why are ChangeSetNames required to start with an Alphabetic character?
+
+                client.create_change_set(
+                    StackName=stack_name,
+                    TemplateBody=codebuild.to_yaml(),
+                    Capabilities=["CAPABILITY_IAM"],
+                    ChangeSetName=change_set_name)
+
+                logging.info("Waiting for change set {}".format(change_set_name))
```

Comment:
Consider new f string format: f"Waiting for change set {change_set_name}"

Reply:
Nice, updated